lms/djangoapps/certificates/api.py

```diff
@@ -949,3 +979,173 @@ def clear_pii_from_certificate_records_for_user(user):
         None
     """
     GeneratedCertificate.objects.filter(user=user).update(name="")
+
+
+def get_cert_history_for_course_id(course_id):
+    """
+    Retrieve all certificate generation history records for the current course.
+    """
+    return CertificateGenerationHistory.objects.filter(course_id=course_id)
+
+
+def get_cert_generation_config():
```

Comment:
Nit:The name suggests that we're getting a full configuration object, but we're really just checking if the current cert generation config is enabled, maybe this should be named something like "is_cert_generation_enabled"?